cmd/mattermost/commands/channel.go

```diff
@@ -549,7 +549,7 @@ func renameChannelCmdF(command *cobra.Command, args []string) error {
 
 	_, errch := a.RenameChannel(channel, newChannelName, newDisplayName)
 	if errch != nil {
-		return errors.New("Error in updating channel from " + channel.Name + " to " + newChannelName + err.Error())
+		return errors.New("Error in updating channel from " + channel.Name + " to " + newChannelName + ": " + errch.Error())
```

Comment:
why `errors.New` vs. `errors.Wrap` here?

Reply:
I changed the line to format the output properly, and to fix a small error, so I maintained the rest of the code unchanged.

Anyway I agree with the change you proposed, so I change it there and in other place in the same file.